apps/docs/pages/guides/functions/local-development.mdx

```diff
@@ -66,7 +66,7 @@ The `functions serve` command has hot-reloading capabilities. It will watch for
 While serving your local Edge Function, you can invoke it using curl:
 
 ```bash
-curl --request POST 'http://localhost:54321/functions/v1/function-name' \
+curl --request POST 'http://host.docker.internal:54321/functions/v1/function-name' \
```

Comment:
Hey @AnmAtAnm! Are you running `curl` inside a container? I think the typical workflow is making that call from the host.

Reply:
Reading the discussion you linked, I think I understand the issue a bit better. When making the request from inside an Edge Function, you should use the built in `SUPABASE_URL` environment variable which will automatically resolve to the correct API URL both locally and in the cloud.

This guide is referring to calling `curl` from the host (not inside an edge function/container), so I don't think we want to make this change.